[Fix #27119] Update PKCE example code on with-nextjs example (#27184)
supabase/supabase: apps/docs/content/guides/getting-started/tutorials/with-nextjs.mdx

Upon sign up, an email will be sent to the user with a unique hash code. This key can then be sent back to the application for verification, thus authenticating the user. The method is referred to as Proof Key for Code Exchange (PKCE). As we are employing PKCE in our authentication flow, it is necessary to create a route handler responsible for exchanging the code for a session.

> Before we proceed, Let's change the Auth confirmation path
> If you have email confirmation turned on (the default), a new user will receive an email confirmation after signing up.
> 
> Change the email template to support a server-side authentication flow following these steps:

Reviewer: ```suggestion
Before we proceed, let's change the email template to support sending a PKCE code:
```
